JellyfinPlayer.xcodeproj/project.pbxproj

```diff
@@ -535,7 +539,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 32;
 				DEVELOPMENT_ASSET_PATHS = "";
-				DEVELOPMENT_TEAM = 9R8RREG67J;
+				DEVELOPMENT_TEAM = 4BHXT8RHFR;
```

Comment:
Please change the development team back to 9R8RREG67J before opening a PR. thanks!

Reply:
oh sorry
I keep forgetting :(

Reply:
its all good :)